Saritasa.Tools.sln

```diff
@@ -32,7 +32,9 @@ Project("{2150E333-8FDC-42A3-9474-1A3956D46DE8}") = "Solution Items", "Solution
 		.editorconfig = .editorconfig
 	EndProjectSection
 EndProject
-Project("{FAE04EC0-301F-11D3-BF4B-00C04F79EFBC}") = "Saritasa.Tools.EntityFrameworkCore", "src\Saritasa.Tools.EntityFrameworkCore\Saritasa.Tools.EntityFrameworkCore.csproj", "{42B411A6-8F91-439F-A006-00196ECF1DA0}"
+Project("{9A19103F-16F7-4668-BE54-9A1E7A4F7556}") = "Saritasa.Tools.EntityFrameworkCore", "src\Saritasa.Tools.EntityFrameworkCore\Saritasa.Tools.EntityFrameworkCore.csproj", "{42B411A6-8F91-439F-A006-00196ECF1DA0}"
```

Comment:
Not sure if `Saritasa.Tools.EntityFrameworkCore` is relevant anymore?
After discussion with @krasninja , it's probably better to use `Saritasa.Tools.EntityFrameworkCore` name instead of `Saritasa.Tools.EFCore`. Could you please update it?